packages/deployer/src/infrastructure/index.ts

```diff
@@ -80,9 +88,9 @@ async function deploy(airnodeIdShort: string, stage: string, region: string, con
 
   command = `terraform apply -var="aws_region=${region}" -var="airnode_id_short=${airnodeIdShort}" -var="stage=${stage}" -var="configuration_file=${path.resolve(
     configPath
-  )}" -var="secrets_file=${path.resolve(
-    secretsPath
-  )}" -var="handler_file=${handlerFile}" -auto-approve -input=false -no-color`;
+  )}" -var="secrets_file=${path.resolve(secretsPath)}" -var="handler_file=${handlerFile}" -var='api_keys=[${apiKeys
```

Comment:
I think this becomes harder to follow when we're doing multi-line interpolation. Maybe assign this to a variable above?